docs/tutorials/creating-a-task-graph.rst

```diff
@@ -136,16 +136,16 @@ comments for explanations):
 
 .. code-block:: python
 
-   from voluptuous import Optional, Required
-
-   from taskgraph.transforms.base import TransformSequence
+   from typing import Optional
    from taskgraph.util.schema import Schema
+   from taskgraph.transforms.base import TransformSequence
+
+   # Define the schema using Schema base class for better type checking and performance.
```

Comment:
nit: I'd cut out `for better type checking and performance` here. The user reading this won't know what you're comparing against